vip-cache-manager/vip-cache-manager.php

```diff
@@ -26,45 +36,43 @@ function __construct() {
 		add_action( 'init', array( $this, 'init' ) );
 	}
 
-	function init() {
+	public function init() {
 		if ( is_super_admin() && isset( $_GET['cm_purge_all'] ) && check_admin_referer( 'manual_purge' ) ) {
```

Comment:
For single-site WordPress, `is_super_admin()` uses the `delete_users` capability (let's set aside that this function even existing in single-site is just silly). `manage_options` strikes me as a more-appropriate single-site capability to check. Thoughts?